child_sync_wp/models/child_compassion.py

```diff
@@ -128,9 +127,9 @@ def _create_diverse_children_pool(self, take):
         })
         try:
             global_pool.country_mix()
-        except Exception, e:
+        except Exception as e:
             logger.exception("The country-aware children selection failed, "
-                             "falling back to rich mix. %s", e.message)
+                             f"falling back to rich mix. {e}")
```

Comment:
Even better, use the real stacktrace by using the parameter from the logging method.
```suggestion
                             "falling back to rich mix.", exc_info=True)
```